download.py

```diff
@@ -11,35 +11,37 @@
 HASH_BLOCK_SIZE = 65536
 
 
-def get_filename(headers):
+def get_filename(headers: list) -> str:
     """ Retrieve a filename from a request headers via Content-Disposition
     """
     content_disp = [x for x in headers if x[0] == 'Content-Disposition'][0][1]
     raw_filename = [x for x in content_disp.split(';') if x.startswith('filename=')][0]
     return raw_filename.replace('filename=', '').replace('"', '')
 
 
-def gdrive_download(gdrive_id, path):
+def gdrive_download(gdrive_id: str, path: str) -> None:
```

Comment:
This functionality was inherited from upstream and IIRC it's not being used anywhere in this project. 

Personally, I haven't verified if it actually works either, and if possible, I'd like to remove this functionality all together. But since users might be using it in their local fixes, I guess we can't really do that.

Reply:
I have come across a number of functions that I am convinced are not being used. I left them in and updated them anyway, as it wasn't in the scope of this PR.

Some stuff could be broken up, like certain functions of the `utils.py`.. but then again some users may need to update their local fixes.

Reply:
For this specifically, if it can be proved that the google drive functionality does not work, then that would warrant removing it. Of course, it doesn't have to be addressed in this PR.

In a future separate PR, we can definitely evaluate removing some of them too. Or like, wait until the upcoming new year for the cleaning 😄